assets/scss/vue-scrollbar.scss

```diff
@@ -0,0 +1,46 @@
+.vue-scrollbar-transition, .vue-scrollbar__scrollbar-vertical, .vue-scrollbar__scrollbar-horizontal {
+  transition: all 0.1s ease;
+}
+.vue-scrollbar-transition--scrollbar {
+  transition: opacity 0.1s linear;
```

Comment:
I think we should make the transition a little bit longer 0.3 looks much more smooth.